src/main/java/at/hannibal2/skyhanni/config/features/chat/ChatConfig.java

```diff
@@ -1,16 +1,18 @@
 package at.hannibal2.skyhanni.config.features.chat;
 
-import at.hannibal2.skyhanni.config.FeatureToggle;
+import java.util.ArrayList;
```

Comment:
the import changes here are against our code formatting patterns. my intellij auto reformats them back to the original when using your changes. please revert those changes here and change your formatting settings accordingly